src/EnergyPlus/ChillerReformulatedEIR.cc

```diff
@@ -116,12 +116,6 @@ namespace ChillerReformulatedEIR {
     // 1. Hydeman, M., P. Sreedharan, N. Webb, and S. Blanc. 2002. "Development and Testing of a Reformulated
     //    Regression-Based Electric Chiller Model". ASHRAE Transactions, HI-02-18-2, Vol 108, Part 2, pp. 1118-1127.
 
-    // chiller part load curve types
-    int const PLR_LeavingCondenserWaterTemperature(1); // Type 1_LeavingCondenserWaterTemperature
-    int const PLR_Lift(2);                             // Type 2_Lift
-
-    Real64 condOutletTempMod(0.0);
```

Comment:
One last module-level variable removed. Now the objects are using their own member variable, as they should. This causes a few diffs as indicated in a previous comment.

Reply:
:+1: